standard/standard-library.md

```diff
@@ -628,4 +634,27 @@ namespace System.Runtime.CompilerServices
         public T GetResult ();
     }
 }
+
+namespace System
+{
+    public interface IFormattable
+    {
+    }
+}
+
+
+namespace System
+{
+    public class FormattableString : IFormattable
+    {
+    }
+}
+
+namespace System.Runtime.CompilerServices
+{
+    public static class FormattableStringFactory
+    {
+        public static FormattableString Create (string format, params object[] arguments);
```

Comment:
Please can we remove the space between `Create` and `(`?

Reply:
```suggestion
        public static FormattableString Create(string format, params object[] arguments);
```